src/components/Navbar.js

```diff
@@ -125,8 +144,8 @@ const Navbar = ({ toggleDarkMode, mode }) => {
                 <Button color="inherit" component={Link} to="/profile">
                   Profile
                 </Button>
-                <Button color="inherit" component={Link} to="/wishlist">
-                  Wishlist
+                <Button component={Link} to="/wishlist">
+                  <FavoriteIcon />
```

Comment:
Right now this is nested under the same `MenuItem` as the profile button above, so it looks like this:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/87777fdf-2267-4481-a7b6-49e5337cb7cd)

That doesn't look quite right to me. If we want it to show up in its own row, we need to set it in its own menuitem in this array:

```suggestion
                <MenuItem key="wishlist" onClick={handleCloseUserMenu}>
                <Button component={Link} to="/wishlist">
                  <FavoriteIcon />
                </Button>
              </MenuItem>,
```

Reply:
done!

Reply:
Looks good! 

Do we intentionally want only the wishlist to be an icon button in the hamburger menu button for mobile? I think it would be better to keep it consistent, either have the whole hamburger menu be words or whole hamburger menu be icons. But I'm not super-design-minded, so I'm happy to proceed however you prefer!

Reply:
@akosasante thanks for the advice, Akos!! I totally agree with you that it´s better to stick to the same pattern, I am a desastrous designer haha so I appreciate any your idea and suggestion!❤️